reports/Virginia.md

```diff
@@ -101,6 +101,20 @@ id: va-richmond-4
 
 * https://twitter.com/BeQueerDoCrime/status/1268648919623442432
 
+
+### Police SUV pushes through protesters, knocks over protester on bike | (believed to be) June 13th
```

Comment:
This seems very similar to the other SUV incident we decided to ignore. I'm borderline on including it

Reply:
I personally added it because as discussed with the other incident, this one shows the SUV moving through the protesters without much of a care for what's in front of it. 

The other incident had the car easing through until given space and trying to move around them.

Reply:
I think that's fair - but on the other hand, the protestors are actively engaging with the car. Maybe another pair of eyes from @maxspencer, @jmc91, @rainmaker would be helpful

Reply:
Someone posted an issue with this exact incident, so I think you might be right to include it